restapi/src/main/java/io/stargate/web/restapi/models/RowAdd.java

```diff
@@ -17,8 +17,12 @@
 
 import io.swagger.annotations.ApiModelProperty;
 import java.util.List;
+import javax.validation.Valid;
+import javax.validation.constraints.NotEmpty;
 
 public class RowAdd {
+  @NotEmpty(message = "Columns cannot be null or empty")
```

Comment:
@mpenick is there any case when adding a row with empty values would work? I guess not..

Reply:
@ivansenic Currently if the columns are empty if error out as below. So I added not empty check for the columns.

curl --location --request POST '127.0.0.2:8082/v1/keyspaces/myworld/tables/mytable/rows' --header "X-Cassandra-Token: $AUTH_TOKEN" \
--header 'Content-Type: application/json' \
--data '{
    "columns":[] }'
{"description":"Bad request: Invalid WHERE clause: DML over a range of partitions are not supported","code":400}